pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -88,6 +88,13 @@ type ReplsetMemberStatus struct {
 	Version string `json:"version,omitempty"`
 }
 
+type MongosStatus struct {
+	Size    int      `json:"size"`
+	Ready   int      `json:"ready"`
+	Status  AppState `json:"status,omitempty"`
+	Message string   `json:"message,omitempty"`
+}
+
```

Comment:
can we also modify CRD a bit, similar to PXC
https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/crd.yaml#L55-L58

Reply:
Will be done in next release